README.md

```diff
@@ -21,9 +21,14 @@ A list of hands-on opportunites (not jobs) in quantum computing
 ## Programs
 | Name | Link | Organization | Date | Status | Feedback |
 | :--: | :--: | :--: | :--: | :--: | :--: |
+| Los Alamos Quantum Computing Summer School| [Info Page](https://www.lanl.gov/projects/national-security-education-center/information-science-technology/summer-schools/quantumcomputing/index.php) | Information Science and Technology Institute (ISTI) | Summer (10 weeks) | ![closed status][closed] | [![FB][feedback]](FEEDBACK.md#los-alamos-quantum-computing-summer-school) |
 | QCSYS | [Info Page](https://uwaterloo.ca/institute-for-quantum-computing/programs/qcsys) | University of Waterloo IQC | August | ![closed status][closed] | [![FB][feedback]](FEEDBACK.md#qcsys-university-of-waterloo-iqc) |
+| QOSF Mentorship program | [Info Page](https://qosf.org/qc_mentorship/) | Quantum Open Source Foundation | Program runs in batches | ![closed status][closed] | [![FB][feedback]](FEEDBACK.md#qosf-mentorship-program) |
+| Summer School on Machine Learning and Big Data with Quantum Computing | [Info Page](https://smbq2020.dcc.fc.up.pt/) | University of Porto | September 7-8 2020 | ![open status][open] | [![FB][feedback]](FEEDBACK.md#SMBQ2020) |
```

Comment:
The link to FEEDBACK is wrong. Try going to FEEDBACK.md and clicking on the link button next to the header, then copypasting the last bit of the url that pops up in your browser